### PR TITLE
feat(EMI-2026): add curators' pick and increased interest signals

### PR DESCRIPTION
**Pull request by rquartararo**
The type of this PR is: **feature**

This PR solves [EMI-2026]

### Description

This adds the Curators' Pick and Increased Interest Signals to the art cards and artwork details pages. 

Screenshot 2024-09-15 at 11 18 24 PM
Screenshot 2024-09-15 at 11 19 06 PM
Screenshot 2024-09-15 at 11 19 30 PM
Screenshot 2024-09-16 at 2 11 12 PM

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[EMI-2026]: https://artsyproduct.atlassian.net/browse/EMI-2026?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/Artwork/Details/Details.tsx [hunk 1]
```diff
@@ -193,35 +192,72 @@ const SaleInfoLine: React.FC<SaleInfoLineProps> = props => {
...
+        px={0.5}
+        alignSelf="flex-start"
+        borderRadius={3}
+      >
+        Limited-time Offer
+      </Text>
+    )
+  }
+
+  if (primaryLabel === "INCREASED_INTEREST" && !hideSignals) {
```
[the remark this answers is not in the document]
Comment (oxaudo) [reply src/Components/Artwork/Details/Details.tsx:225]: My understanding was that we were talking about surpassing specific signals on the page/rail so `hideSignals` is not a boolean but an array of signals to suppress. But maybe I misunderstood the conversation. @erikdstock - what was the idea?

**Review by oxaudo — COMMENTED** [review]
I have a question about how this change is going to play with release plan? Should this stuff be behind feature flag? Otherwise looks good to me!

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarSignal.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+interface ArtworkSidebarSignalProps {
+  artwork: ArtworkSidebarSignal_artwork$key
+}
+
+export const ArtworkSidebarSignal: React.FC<ArtworkSidebarSignalProps> = ({
+  artwork,
+}) => {
+  const data = useFragment(ArtworkSidebarSignalFragmentContainer, artwork)
+  const { t } = useTranslation()
```
[the remark this answers is not in the document]
Comment (damassi) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarSignal.tsx:18]: Ah, we're not using i18n any more, so this should be removed

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarSignal.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+          </RouterLink>
+        </Stack>
+      </Flex>
+    )
+  }
+
+  return null
+}
+
+const ArtworkSidebarSignalFragmentContainer = graphql`
```
Comment (damassi) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarSignal.tsx:87]: We shouldn't be writing legacy relay fragment containers anymore but rather using relay hooks

Comment (damassi) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarSignal.tsx:87]: ahh hah, yep 😄 just realized that

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarSignal.tsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+export const ArtworkSidebarSignal: React.FC<ArtworkSidebarSignalProps> = ({
+  artwork,
+}) => {
+  const data = useFragment(artworkSidebarSignalFragment, artwork)
+
+  const isShowingNow = data.collectorSignals?.runningShow ?? false
+  const shouldRenderCuratorsPick =
+    data.collectorSignals?.primaryLabel === "CURATORS_PICK"
+  const shouldRenderIncreasedInterest =
+    data.collectorSignals?.primaryLabel === "INCREASED_INTEREST"
```
Comment (erikdstock) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarSignal.tsx:22]: We might be able to use a switch statement or conditional here. What it if was just
```tsx
const primaryLabel = data.collectorSignals?.primaryLabel
if (primaryLabel === "CURATORS_PICK") {
  return (<Flex alignItems="top" my={4} data-testid="curators_pick">
    <VerifiedIcon color="black100" width={18} mr={1} mt={0.5} />
}
```
etc. A `switch` would be another option, but the need to account for showing now (should this go in `primaryLabel`?) might make that a little uglier.

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarSignal.tsx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    collectorSignals {
+      primaryLabel
+      runningShow {
+        name
+        href
+        startAt
+        endAt
+      }
+    }
+  }
```
Comment (erikdstock) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarSignal.tsx:100]: Any need to use the `ignore` arg here?

File: src/Components/Artwork/Details/Details.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (erikdstock) [reply src/Components/Artwork/Details/Details.tsx:225]: Sorry for the late response! The argument is `ignore`, from our schema [here](https://github.com/artsy/force/blob/79027b9dfed42d2737e2154f5f2fbccbcd60ecbf/data/schema.graphql#L4458-L4461): 
`artwork(id: 'foo') { collectorSignals { primaryLabel(ignore: [INCREASED_INTEREST]) } }

**Review by erikdstock — COMMENTED** [review]
Lookin good, a few minor comments!

**Pull request merged**